### PR TITLE
[18.0][MIG] document_page_work_instruction: Migration to 18.0

### PR DESCRIPTION
**Pull request by Kimkhoi3010**
Depend on : [#504](https://github.com/OCA/knowledge/pull/504)

### CODE REVIEW

File: document_page_work_instruction/readme/CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
+
+The migration of this module from 16.0 to 18.0 was financially supported by:
+
+- Camptocamp.
```
Comment (ajaniszewska-dev) [inline document_page_work_instruction/readme/CREDITS.md]:
```suggestion
- Camptocamp
```

nitpicking

**Review by ajaniszewska-dev — APPROVED** [review]

File: document_page_work_instruction/readme/CREDITS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
+
```
Comment (max3903) [inline document_page_work_instruction/readme/CREDITS.md]:
```suggestion
```

File: document_page_work_instruction/static/description/icon.png
Comment (max3903) [inline document_page_work_instruction/static/description/icon.png:1]: @Kimkhoi3010 Please update the icon.

File: document_page_work_instruction/readme/CREDITS.md (hunk 2 above)
Comment (sebalix) [reply document_page_work_instruction/readme/CREDITS.md]: @Kimkhoi3010 can you address this point please?

**Review by max3903 — APPROVED** [review]

**Pull request merged**